mass_editing.py

```diff
@@ -99,6 +99,21 @@ class MassEditFields(ModelSQL):
     field = fields.Many2One('ir.model.field', 'Field', required=True,
         ondelete='CASCADE')
 
+    @classmethod
+    def validate(cls, fields):
+        super().validate(fields)
+        for _field in fields:
+            _field.check_field()
+
+    def check_field(self):
+        Model = Pool().get(self.field.model.model)
+
+        _field = Model._fields.get(self.field.name)
+        print(self.field.name, _field)
```

Comment:
This print should be removed.

Reply:
1ba7bdee1f26e4dd57197db15920ff3fb79033ee